configure.ac

```diff
@@ -234,7 +234,7 @@ AS_IF([test "x$with_docs_only" == xyes],
      AC_ARG_ENABLE([examples],
                    [AC_HELP_STRING([--enable-examples],
                                    [Enable examples build])],
-                   [AM_CONDITIONAL([HAVE_EXAMPLES], [true])],
+                   [AM_CONDITIONAL([HAVE_EXAMPLES], [test $enable_examples == yes])],
```

Comment:
need to do "X$examples" == "Xyes"
also let's add --enable-examples to ./contrib/configure-devel?

Reply:
fixed/added